### PR TITLE
feat(zoolander):homepage/subnav/mobile

### PR DESCRIPTION
**Pull request by amberRrucker**
Fixes for subnav/icon color and homepage additions.

### CODE REVIEW

File: styleguide/derek/examples/_homepage.html [hunk 1]
```diff
@@ -223,40 +231,48 @@ <h2 class="right">FANATICAL SUPPORT WhereVER YOU ARE, DaY OR NIGHT, WE HAVE YOU
...
           <div class="col-sm-2 center-block">
-            <i class="rsweb gen-support white center"></i>
+            <i class="rsweb gen-support center"></i>
           </div>
         </div>
       </div>
           <div class="row">
             <div class="col-md-6">
-              <p class="white">Our helpful and always ready experts are a click or call away. <br/>Anytime. Anywhere.</p>
+              <p class="">Our helpful and always ready experts are a click or call away. <br/>Anytime. Anywhere.</p>
```
Comment (pobtastic) [inline styleguide/derek/examples/_homepage.html]: There's an empty class element here, should it be removed?

File: styleguide/derek/examples/_homepage.html [hunk 2]
```diff
@@ -223,40 +231,48 @@ <h2 class="right">FANATICAL SUPPORT WhereVER YOU ARE, DaY OR NIGHT, WE HAVE YOU
...
+        </div>
+        <div class="col-xs-2">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/six-flags-logo.png"/>
+        </div>
+        <div class="col-xs-2">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/ksLogo.png"/>
+        </div>
+        <div class="col-xs-2">
+          <img src="https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/logos/razorfish-logo.png"/>
+        </div>
```
Comment (pobtastic) [inline styleguide/derek/examples/_homepage.html]: @ejchiefs Can we have a ticket in the backlog for adding alt to all image tags for accessibility 👍

File: styleguide/derek/examples/_homepage.html (hunk 1 above)
Comment (amberRrucker) [inline styleguide/derek/examples/_homepage.html]: I'm not sure if its going to be white or a dark background or not yet. The homepage is in progress.

File: styleguide/derek/examples/_homepage.html (hunk 2 above)
Comment (amberRrucker) [inline styleguide/derek/examples/_homepage.html]: @ejchiefs would that be a ticket for SEO?

**Comment by aedile** [comment]
Looping around for documentation purposes.  This had a 👍  from both me and @pobtastic.

**Pull request merged**